taskcluster.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	queue "github.com/taskcluster/taskcluster-client-go/queue"
+	"log"
+)
+
+func getTaskScopes(taskId string) []string {
+	// We do not need auth for this operation
+	q := queue.New("", "")
+	q.Authenticate = false
+
+	task, callSummary := q.Task(taskId)
+	if callSummary.Error != nil {
+		log.Fatalf("Could not fetch taskcluster task '%s' : %s",
```

Comment:
nit: IMO I would return this error and let the main control flow deal with fatal'ing out